### DISCUSSION TITLE
Exact instead of rounded off values in progress cell

### DISCUSSION THREAD

**Discussion by n-srinidhi** [h2oai/wave#2136] [Q&A] [answered]
Hi! I am using `cell_type=ui.progress_table_cell_type()` in a table to display progress values. I would like it to display the exact value I give For Eg: I have given the value as 0.081 for displaying 8.81%, but it rounds off the value to 9 while displaying. Any way I can retain the exact value and display it as 8.81%?

![image](https://github.com/h2oai/wave/assets/117831190/7de150da-ada8-4a91-9143-9285c2ef3569)

**Comment by vopani** [comment, marked as answer]
https://github.com/h2oai/wave/issues/824

**Reply by n-srinidhi** [reply]
Any idea when this will be available?
